include/diaspora.php

```diff
@@ -932,18 +932,20 @@ private static function post_allow($importer, $contact, $is_comment = false) {
 			logger("defining user ".$contact["nick"]." as friend");
 		}
 
-		// We don't seem to like that person
+		// Is this contact wanted?
```

Comment:
Personally I don't have nothing against "we do this here" but `// We don't seem to like that person` is a declarative sentence which is misleading and at this place not true. We check if we like the contact :-)

I like to have explanatory comments above conditions for two possibilities.
1.) If the condition and its variables are not self explanatory (e.g. `$is_comment` is self explanatory, `$importer["uid"] == 0` would need an explanation)
2.) Why we need to check for this condition (This would make the code - and the logic of friendica internals - more understandable to newcomers and a developer who is not familiar with this code part can jump to the position he/she is interested in without reading the whole condition)

After 2 1/2 years participating in friendica development and reading much code there are many parts where I no knowledge about. Not because I don't understand the code but because I get no knowledge about the background why something has to happen. To give an example (have a look at the original comment of line 941 of the original file):
The comment is: `// Is it a post to a community? That's good` As a newcomer I would ask myself why it is good if a post is a post to a community account.